### DISCUSSION TITLE
How to get the stackblitz postcss JIT demo to remove rules?

### DISCUSSION THREAD

**Discussion by roblevintennis** [argyleink/open-props#118] [Q&A] [answered]
I'm using a fork of the stackblitze example, and I believe I've removed everything but one usage of open props custom properties, I run `npm run build` and I look at generated .css file and it appears to still have everything.

Lmk if I'm doing something wrong with stackblitz (I don't use this much) but I would think I'm following correct steps to see the JIT rule removal ¯\_(ツ)_/¯

https://stackblitz.com/edit/jit-open-props-szgzi4?file=dist/assets/index.540a222e.css

Preview:
![image](https://user-images.githubusercontent.com/142403/145440216-c7dbd619-0b22-4974-8d87-ea488cb674ad.png)

**Comment by argyleink** [comment, marked as answer]
![Screen Shot 2021-12-09 at 9 06 41 AM](https://user-images.githubusercontent.com/1134620/145442758-bbfec9c1-eee8-4b23-b520-fa3a3774a615.png)

gotta remove both of these imports.

**Reply by roblevintennis** [reply]
Ah I see now -- your requiring it in the `postcss.config.js` already hadn't noticed that. Thanks.

**Comment by argyleink** [comment]
it's not about removal of rules, it's about using them and having them automatically added on use 👍🏻

**Reply by roblevintennis** [reply]
Actually, now that I've removed the imports as you've mentioned the result is as I expected:

![image](https://user-images.githubusercontent.com/142403/145448415-b569cc03-6148-4419-a846-139c4172ea17.png)

Nifty 🎉

**Reply by jasikpark** [reply]
yeah, i think making it more explicitly clear that it's about automatically generating what you need & that you don't need to import *anything* would be helpful! 😁

**Reply by roblevintennis** [reply]
Now that you've helped me get it working (🙏 @argyleink) I'm wondering in a large project how I'd leverage pointing jit to all my various css files e.g. can I take:

`{files: ['./props.css']}` from the example and do glob'ing like:

`{files: ['./**/vendor/*.css', './libs/**/*.css', 'path/to/one-off.css']}`?

**Reply by argyleink** [reply]
that's a question for the jit-props plugin!

but i can partially answer here: glob import is not supported. there's def room for more options and stronger features in the import mechanisms of that plugin 👍🏻
